### ISSUE TITLE
Documentation states you can bypass launchpad with flags, not working

### ISSUE THREAD

**Issue by xjdesigns** [cypress-io/cypress-documentation#5811]
### Subject

examples, guides

### Description

Upon review of Cypress documentation under the "cypress app" section if you pass the flag of "--browser", "--component", or "--e2e" you can bypass the launchpad and go straight to the spec page. Here is the wording and a link.
```
Once you get comfortable with Cypress you might find you don't need the Launchpad any more. In this case [you can run cypress open with the --browser and --component or --e2e options](https://docs.cypress.io/guides/guides/command-line#cypress-open) to go straight to the Specs page.
```
[Docs](https://docs.cypress.io/guides/core-concepts/cypress-app)

I saw a thread from Nov 2023 noting this was not possible. After testing using these args in the command this is true and launchpad still opens. Would be good to update the documentation if this is not a bug and proper to avoid more people spending time trying to debug something documented which is not possible.

![Screenshot 2024-04-19 at 3 50 12 PM](https://github.com/cypress-io/cypress-documentation/assets/7092230/5810602a-c9c2-40ea-bedf-c707d7d22243)

**Comment by MikeMcC399** [comment]
@xjdesigns 

You are right that the description is open to interpretation and could be made clearer.

The wording does not explicitly says that the Cypress Launchpad is **not** started. Using the command line options means you don't have to click your way through to getting the Cypress Runner to start, instead it starts together with the Cypress Launchpad.

**Comment by jennifer-shehane** [comment]
We can't skip entirely not opening the Launchpad so that we can handle error states and showing views in those error states (like an incorrect config) without crashing the entire experience or taking you to the terminal for that error.
